server/tests/test_api.py

```diff
@@ -147,3 +147,12 @@ def test_log_in_csrf_failure(client: FlaskClient) -> None:
     """Test post request without proper headers results in 403."""
     res = client.post("/login", json={"access_token": "token"})
     assert res.status_code == 403
+
+
+def test_logout(client: FlaskClient, mocker: MockerFixture) -> None:
+    """Test logout returns data successfully."""
+    mocker.patch("flask_login.logout_user")
+    app.config["LOGIN_DISABLED"] = True
```

Comment:
what does this do?

Reply:
There's a `@login_required` decorator from `flask_login`. It's difficult to mock that, so the suggested approach is to turn off `LOGIN` while testing. 
```
It can be convenient to globally turn off authentication when unit testing.
To enable this, if the application configuration variable LOGIN_DISABLED is set to True,
this decorator will be ignored.
```

https://flask-login.readthedocs.io/en/latest/#protecting-views